openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -249,6 +252,9 @@
 .pl-0 {
   padding-left: 0px;
 }
+.pl-1 {
+  padding-left: 16px;
+}
```

Comment:
we already have `p-l-md` class which provides padding-left of 16px, can we use that?
https://github.com/open-metadata/OpenMetadata/blob/339abc5bf33aa3f6b56166145d58fdca0119ed92/openmetadata-ui/src/main/resources/ui/src/styles/spacing.less#L240

Reply:
Removed it as we are not using it. Thanks for catching it